docs/10.0/api/sidebar.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs');
 const path = require('path');
-const { getBuildDocsVersion, getLatestVersion } = require('../../.vuepress/helpers');
+const { getEnvDocsVersion, getLatestVersion } = require('../../.vuepress/helpers');
```

Comment:
This would probably be more readable if each imported prop would be in its own line?

Reply:
I think that it's not needed as it's just one line with two imports.